modules/v2x/v2x_proxy/proto_adapter/proto_adapter.cc

```diff
@@ -0,0 +1,230 @@
+/******************************************************************************
+ * Copyright 2020 The Apollo Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *****************************************************************************/
+
+#include "modules/v2x/v2x_proxy/proto_adapter/proto_adapter.h"
+
+#include <Eigen/Core>
+#include <Eigen/Geometry>
+
+#include "cyber/cyber.h"
+
+namespace v2x {
+namespace proxy {
+
+OSLightype AdapterLightTypeObu2Sys(int32_t type) {
+  switch (type) {
+    case 2:
+      return OSLightype::SingleTrafficLight_Type_LEFT;
+    case 3:
+      return OSLightype::SingleTrafficLight_Type_RIGHT;
+    case 4:
+      return OSLightype::SingleTrafficLight_Type_U_TURN;
+    case 1:
+    default:
+      return OSLightype::SingleTrafficLight_Type_STRAIGHT;
+  }
+}
+
+bool AdapterLightObu2Sys(const OBULight &obu_light,
```

Comment:
And what about putting all the functions as static member functions in one class named `ProtoAdapter` ?